pkg/setting/setting_test.go

```diff
@@ -96,7 +86,7 @@ func TestLoadingSettings(t *testing.T) {
 			err = cfg.Load(&CommandLineArgs{HomePath: "../../"})
 			So(err, ShouldBeNil)
 
-			So(appliedEnvOverrides, ShouldContain, "GF_DATABASE_URL=mysql://user:-redacted-@localhost:3306/database")
+			So(appliedEnvOverrides, ShouldContain, "GF_DATABASE_URL=mysql://user:xxxxx@localhost:3306/database")
```

Comment:
For the sake of security (and some other reasons), now we rely on Go standard library function to redact URLs, which uses `xxxxx` as placeholder, which I guess it's fine.